editor/create_dialog.cpp

```diff
@@ -37,6 +37,7 @@
 #include "editor_node.h"
 #include "editor_settings.h"
 #include "scene/gui/box_container.h"
+#include "editor_feature_profile.h"
```

Comment:
Includes should be ordered alphabetically, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes